app/recurring_job.go

```diff
@@ -431,6 +472,22 @@ func (job *Job) doSnapshotCleanup(backupDone bool) (err error) {
 	return nil
 }
 
+func (job *Job) eventCreate(eventType, eventReason, message string) error {
+	jobName, found := job.labels[types.RecurringJobLabel]
+	if !found || jobName == "" {
+		return fmt.Errorf("missing RecurringJob label")
+	}
+	recurringJob, err := job.lhClient.LonghornV1beta2().RecurringJobs(job.namespace).Get(context.TODO(), jobName, metav1.GetOptions{})
+	if err != nil {
+		job.logger.WithError(err).Errorf("Failed to get recurring job %v.", jobName)
```

Comment:
No need to log here, but rather let the caller to deal with the log if needed.

Reply:
Removed. Thanks